File: src/backend/api_integration.js
```javascript
const axios = require('axios');
const { RSVP } = require('./database.js');

const importRSVPList = async () => {
  try {
    const response = await axios.get('https://api.partiful.com/rsvp');
    const rsvpList = response.data;

    rsvpList.forEach(async (rsvp) => {
      const newRSVP = new RSVP({
        name: rsvp.name,
        gender: rsvp.gender,
        email: rsvp.email
      });

      await newRSVP.save();
    });

    console.log('RSVP list imported successfully.');
  } catch (error) {
    console.error('Error importing RSVP list:', error);
  }
};

module.exports = { importRSVPList };
```

File: src/api_integration.py
```python
import requests
from src.database import RSVP

def importRSVPList():
    # Define the Partiful API endpoint
    partiful_api_endpoint = "https://api.partiful.com/rsvp"

    # Send a GET request to the Partiful API
    response = requests.get(partiful_api_endpoint)

    # If the request was successful
    if response.status_code == 200:
        # Parse the JSON response
        rsvp_list = response.json()

        # For each RSVP in the list
        for rsvp in rsvp_list:
            # Create a new RSVP object
            new_rsvp = RSVP(name=rsvp['name'], gender=rsvp['gender'], email=rsvp['email'])

            # Save the new RSVP to the database
            new_rsvp.save()

        # Return the list of RSVPs
        return rsvp_list

    # If the request was not successful, raise an exception
    else:
        raise Exception("Failed to import RSVP list from Partiful API")
```